charts/k8s-watcher/templates/network-mapper/serviceaccount.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
This file is confusing, it's not creating a service account but creating the Bindings to the existing k8s watcher service account

Reply:
Will rename to roles-binding.yaml